README.md

```diff
@@ -11,7 +11,7 @@ To login, use the environment variables value defined in:
 
 Default Version: v8.3.2
 
-[![Deploy](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy)
+[![Deploy](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy?source=https://github.com/Scalingo/grafana-scalingo)
```

Comment:
With the deployment of the new dashboard, the new URL is:

```suggestion
[![Deploy](https://cdn.scalingo.com/deploy/button.svg)](https://dashboard.scalingo.com/create/app?source=https://github.com/Scalingo/grafana-scalingo)
```

I would prefer we use this new path even though the `my.scalingo.com` one still work.